src/pages/planning-applications/[id]/index.tsx

```diff
@@ -33,10 +33,14 @@ export async function getStaticProps(context: any) {
 export async function getStaticPaths() {
   const data = await getActiveApplications();
 
-  return {
-    paths: data.map((doc: any) => ({ params: { data: doc, id: doc._id } })),
-    fallback: "blocking",
-  };
+  if (!data || data.length === 0) {
+    return { paths: [], fallback: "blocking" };
+  } else {
+    return {
+      paths: data?.map((doc: any) => ({ params: { data: doc, id: doc._id } })),
```

Comment:
you can remove the .?  now as its garenteed to exist right?

Reply:
If I remove breaks the building on yearn build